### DISCUSSION TITLE
Discuss how we can make best use of Discussions

### DISCUSSION THREAD

**Discussion by talltree** [trustoverip/trust-spanning-protocol#8] [General]
Since we're a new ToIP task force and many of us are also new to GitHub Discussions as a tool, this thread is simply to explore and propose the best ways we can use Discussions to advance our mission.

I wanted to start by recommending the [GitHub Discussions Quickstart page](https://docs.github.com/en/discussions) as it has a whole lot of good info and best practices about using Discussions. If you haven't used Discussions before (like me), it has lots of good advice and tips. For example, did you know you can easily [create a poll](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions#about-polls) in any Discussion thread?

**Comment by talltree** [comment]
For the GitHub experts, are there any other steps we need to take to prepare this repo for our use as a task force?

**Reply by sankarshanmukhopadhyay** [reply]
My suggestion would be to decide on a periodic check of the discussion topics to determine the ones ready to be tracked as Issues and hence will have PRs against them. 

The discusion activity dashboard on this project must be quite a thing!

**Comment by Oskar-van-Deventer** [comment]
The purpose of Discussions is that they lead something: at least to some conclusion, or better to some action.

Once ToIP TSPTF is done presenting and discussing all proposals (like Sam Smith's #17) during the next few weeks, the meeting could start going through the list of discussions and determine their status (active/ongoing, stale, unresolved, stuck-without-consensus, ...). Based on this status decision, the meeting could suggest follow-up, e.g. request a Pull-Request or some other action.

One thing that we should get clear, is how do submitters determine whether their point is for Discussions, or for Issues?

**Reply by Oskar-van-Deventer** [reply]
I notice that some of the discussions are rather verbose. Whereas there is nothing wrong about posting a 4000-word discussion item, like #20, it makes the discussion perhaps somewhat less accessible. In contrast, the compact #19 with focused key take-aways and a compact picture really help quick understanding.

**Reply by talltree** [reply]
@Oskar-van-Deventer I have been thinking that once we reach the consolidation stage, we would begin reviewing these discussions to see which points have been resolved/incoporated, and which still remain open issues. The latter would be either converted into issues, or new issues would be filed that could refer back to the discussion.

**Reply by talltree** [reply]
Also, on your point about the verbose discussions, I too want to encourage concise discussions whenever possible. However I recognize certain deep issues might require more lengthy discussion, especially when it needs to be asynchronous. In those cases, I have two strong suggestions for the participants:

1. First, as with good email etiquette, put the word "LONG" in the title of the discussion.
2. Second, if/when there is a specific conclusion or outcome, it would be ideal if whomever started the discussion (and thus has permission to edit the first post) could to add a note to that first post starting with "CONCLUSION" or "OUTCOME" and a link to a final post that summarizes the conclusion or outcome.
